### ISSUE TITLE
Extended result screen isnt exactly obvious for all players

### ISSUE THREAD

**Issue by Lolihunter1337** [ppy/osu#11441]
**Describe the new feature:** 
Accessing the Extended Result screen is "hidden" behind a click on your Result screen card. I feel like telling the player that it exists would be nice? espacially for new players.
**Proposal designs of the feature:** 
I dont really have one but something like "click here to show extra stats" or something

**Comment by peppy** [comment]
Yep, definitely agree. it's very hidden right now, unlike on stable where there's a visible button to show the extended stats.

@arflyte did you have anything in mind for this?

**Comment by Malmik** [comment]
What about pulsating "+" sign somewhere? Preferably in a circle?
